packages/sdk/README.md

```diff
@@ -2,232 +2,65 @@
 
 The Looker SDK for Typescript/Javascript works with Node and browser run-times. The SDK provides a convenient way to communicate with a Looker server's APIs.
 
+This package supports using the Looker SDK in the browser. The [@looker/sdk-node](https://www.npmjs.com/package/@looker/sdk-node) package depends on this package (@looker/sdk) and [@looker/sdk-rtl](https://www.npmjs.com/package/@looker/sdk-rtl).
+
 The SDK uses a plug-in architecture (also known as dependency injection) for initializing that supports run-time specific transports (like `NodeTransport` and `BrowserTransport`) and different approaches for managing API authentication (like `NodeSession`, `BrowserSession`, `ProxySession`, and `CorsSession`).
 
-**DISCLAIMER**: This is a _beta_ version of the Looker SDK, using a completely new code generator developed by Looker. Implementations are still subject to change, but we expect most SDK method calls to work correctly. If you run into problems with the SDK, please feel free to [report an issue](https://github.com/looker-open-source/sdk-codegen/issues), and please indicate which language SDK you're using in the report.
+**DISCLAIMER**: This is a _beta_ version of the Looker SDK. Implementations are still subject to change, but SDK method calls are expected to work correctly. Please [report any issues](https://github.com/looker-open-source/sdk-codegen/issues) encountered, and indicate the SDK language in the report.
 
 ## Getting started
 
-The Looker SDK can be used in a node application in 3 steps:
+The Looker Browser SDK can be used in a browser application in 3 steps:
 
 - install
-- configure
+- authenticate
 - use
 
-### Install the Looker SDK into your node application
-
-Using `npm`:
-
-```bash
-npm install @looker/sdk @looker/sdk-node
-```
+### Install the Looker SDK into your application
 
 Using `yarn`:
 
 ```bash
-yarn add @looker/sdk @looker/sdk-node
+yarn add @looker/sdk
```

Comment:
see previous comment about explicitly mentioning sdk-rtl. Mainly because of linting rules

Reply:
added to both readme installation instructions